run_simple_experiment.py

```diff
@@ -0,0 +1,254 @@
+    print("USERS:", full_df['worker'].nunique())
+    print("ITEMS:", full_df['question'].nunique())
+    print("ANSWERS:", len(full_df))
+
+    if task_type in ['numerical', 'ordinal']:
+        dist_fn, eval_fn = make_numerical_fns(full_df)
+        dist_fn = lambda x, y: abs(x - y)
+        eval_fn_dict = {"diff/range e": eval_fn}
+        dist_fn_dict = {"diff/range d": dist_fn}
+        if task_type == 'numerical':
+            merge_fn = merge_functions.numerical_mean
+        else:
+            merge_fn = merge_functions.numerical_mean_rounded
+
+    elif task_type == 'categorical':
+        eval_fn_dict = {"exact match eval": lambda x, y: 1 if x == y else 0}
+        dist_fn_dict = {"exact match dist": lambda x, y: 0 if x == y else 1}
+        merge_fn = None
+
+    results, scores_all = test_simple_experiment(task_name, 
+                                    SimpleExperiment, 
+                                    task_type, 
+                                    full_df, 
+                                    eval_fn_dict=eval_fn_dict, 
+                                    dist_fn_dict=dist_fn_dict,
+                                    merge_fn=merge_fn,
+                                    run_merge=merge,
+                                    supervised_items=supervised_items)
+    results['supervision_amt'] = supervision_amt
+    results['fold'] = fold
+    results['noise'] = noise
+    results['suffix'] = suffix
+
+    print(scores_all)
+    print(results)
+    if log_dir:
+
+        '''TODO: address when there is more than one dist fn or eval fn so 
+        len(scores_all) > 1 '''
+        assert(len(scores_all) == 1)
+        scores_dict = list(scores_all.items())[0][1]
+
+        for method, scores in scores_dict.items():
+            with open(f'{log_dir}/{method.lower()}_scores_{task_name}_{suffix}.csv', 'w') as file:
+                writer = csv.writer(file)
+                for item, val in scores.items():
+                    writer.writerow([item,val])
+
+
+        filename = f"{log_dir}/results_{task_name}_{suffix}.csv"
+        results.to_csv(filename, mode='w', header=True, index=False)
+
+
+        ''' to fit with the old formatting requirements '''
+        # for idx, row in results.iterrows():
+        #     filename = f"{log_dir}/{row['method'].upper()}/results_{task_type}.csv"
+        #     with open(filename, 'a') as outfile:
+        #         writer = csv.writer(outfile)
+        #         if (os.stat(filename).st_size == 0):
+        #             if task_type == 'categorical':
+        #                 writer.writerow(['task', 'accuracy', 'f1'])
+        #             else:
+        #                 writer.writerow(['task', 'mae', 'mse'])
+        #         if task_type == 'categorical':
+        #             writer.writerow([row['task'], row['accuracy'], row['f1_weighted'], row['f1_macro']])
+        #         else:
+        #             writer.writerow([row['task'], row['mae'], row['mse']])
+
+
+
+main()
```

Comment:
you can use the pattern below to avoid running main in case this module is just getting imported:

if __name__ == "__main__":
    main()